### DISCUSSION TITLE
Failed to deploy Vue project

### DISCUSSION THREAD

**Discussion by marswong** [vercel/community#5137] [Help] [answered]
### Summary

All deployments are failed today, but no detailed error logs, I can successfully build it locally.

![image](https://github.com/vercel/community/assets/10388933/53b4511a-e03e-4f7f-a72f-12fb4f841346)

![image](https://github.com/vercel/community/assets/10388933/cbc8fe70-05db-4ffb-905c-e29bfde95821)

![image](https://github.com/vercel/community/assets/10388933/a8ce7152-215e-42f3-a88d-678707074fbf)

### Steps to Reproduce

just trigger the deployment

**Comment by igmartell** [comment]
I have the exact same issue:

@https://github.com/orgs/vercel/discussions/5139

**Comment by marswong** [comment]
It seems to be an issue of `thread-loader` using Node.js `v18.8.0`: https://github.com/webpack-contrib/thread-loader/issues/191#issuecomment-1728002022. @igmartell maybe you could give it a try.

**Reply by marswong** [reply, marked as answer]
If you are getting into the same issue, please try setting the env `UV_USE_IO_URING=0` in your build command, e.g:

![Screen Shot 2023-12-12 at 16 43 27](https://github.com/vercel/community/assets/10388933/8c7841f3-475e-4ef9-a6e2-dd93ab98d51e)

**Reply by joaomirandas** [reply]
Yep!! This worked well for me here. My builds can now be completed - thanks @marswong

**Reply by igmartell** [reply]
I also made it work this way, but I personally prefer @amyegan ‘s approach because I don’t know what that env variable is doing honestly.(https://github.com/orgs/vercel/discussions/5137#discussioncomment-7821411)

**Reply by marswong** [reply]
@igmartell FYI, `io_uring` is an unstable feature provided by linux kernal v5.11+, which was introduced in libuv's recent versions, adopted in Node.js v18.8.0, but reverted in later versions.

If you can suffer manual deployment, local deployment works well. But if you rely on CI/CD, I cannot find a better way apart from this :)

**Comment by joaomirandas** [comment]
Same issue here, my deployments get stucked in build - never end.

**Comment by amyegan** [comment]
Thanks for reporting this. If you run [`vercel build`](https://vercel.com/docs/cli/build) on your local repo, do you get the same error? If the local build succeeds, try running `vercel deploy --prebuilt` to [deploy from local build](https://vercel.com/docs/cli/deploying-from-cli#deploying-from-local-build-prebuilt).

The issue seems related to https://github.com/vuejs/vue-cli/issues/5898. It could be caused by a recent minor or patch version update in a dependency if specific versions aren't pinned. Or it could have a different source. But if the steps above work for you, then we'll at least have a way to get you unstuck while we figure out what's causing the problem 🙇

**Reply by igmartell** [reply]
If I follow these steps, it is successfully built locally and deployed to Vercel.
Thank you for the workaround.

**Comment by joaomirandas** [comment]
Any updates on this, @amyegan?

My builds never complete, so I'm unable to update anything deployed on Vercel. For your information, using Amplify on AWS, everything works well.

Also, when I downgrade the Node version to 16.x, my builds complete successfully. It's something to do with using Node >= 18.x. I can't pinpoint what exactly.

My builds stops at that stage:
![Captura de Tela 2023-12-11 às 19 37 15](https://github.com/vercel/community/assets/29043431/d12c6387-1a35-414f-9b97-1f2640d21d8f)

**Reply by marswong** [reply]
Have you tried my [comment](https://github.com/orgs/vercel/discussions/5137#discussioncomment-7810398) above? The reason and workaround are included in `thread-loader`'s issue.

**Comment by tonniva** [comment]
Same issue 

Can I wait? Or what should I do?

<img width="1405" alt="Screenshot 2566-12-12 at 11 18 52" src="https://github.com/vercel/community/assets/38358963/15048632-ea8f-4e67-9686-14b811bdcf22">

**Reply by marswong** [reply]
Did you set the env `UV_USE_IO_URING=0`?

**Reply by tonniva** [reply]
Very good bro succeeded .
